vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/DeviceSteps.java

```diff
@@ -76,21 +80,68 @@ public void uploadFileToDevice(String filePath)
      * See <a href="https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton">iOS keys</a> and
      * <a href="https://appium.github.io/java-client/io/appium/java_client/android/nativekey/AndroidKey.html">
      * Android keys</a> for available values
+     * <br>
+     * Example:
+     * <br>
+     * <code>
+     * When I press $key key
+     * </code>
      *
-     * @param key key to press
+     * @param key the key to press
      */
     @When("I press $key key")
```

Comment:
should it be deprecated?

Reply:
I would prefer to use this step if I need to type only one key, it's 1 vs 3 lines

Reply:
so what about moving it to the composite step in this case?

Reply:
End users are expected to use this step extensively, absence of auto-complete or quick docs will hurt their feelings

Reply:
not the reason to do not convert it to composite

Reply:
why is the end-user unfriendliness not the reason?

Reply:
because following this reason all composite steps must be moved to Java

Reply:
so we don't care about user experience and it's ok for us that the step becomes hardly usable due to the absence of autocomplete and docs

Reply:
From my point of view if the step is not deprecated moving into composites will decrease usability, because of absent autocomplete and docs.

I was thinking that composites are for backward compatibility and deprecation, not for "overloading" the steps